vital/types/object_track_set.cxx

```diff
@@ -44,29 +44,52 @@ namespace vital {
 
 typedef std::unique_ptr< track_set_implementation > tsi_uptr;
 
-
 /// Default Constructor
 object_track_set
 ::object_track_set()
   : track_set( tsi_uptr( new simple_track_set_implementation ) )
 {
 }
 
-
 /// Constructor specifying the implementation
 object_track_set
 ::object_track_set( std::unique_ptr<track_set_implementation> impl )
   : track_set( std::move( impl ) )
 {
 }
 
-
 /// Constructor from a vector of tracks
 object_track_set
 ::object_track_set( std::vector< track_sptr > const& tracks )
   : track_set( tsi_uptr( new simple_track_set_implementation( tracks ) ) )
 {
 }
 
+detected_object_sptr& object_track_state::detection()
+{
+  return detection_;
+}
+const detected_object_cptr object_track_state::detection() const
```

Comment:
So, I see separator lines are already missing here :cry:, but please at least keep a blank line between functions. (Feel free to keep *one*, though; removing the multiple blank lines above is okay!)